ops/generate-subgraph.js

```diff
@@ -46,6 +47,8 @@ async function generateSubgraph() {
 	const subgraph = {
 		specVersion: '0.0.1',
 		schema: { file: './schema.graphql' },
+		description: `DAOstack enables the effective governance of self-organizing collectives. Find DAO's members, proposals, votes, and more.`,
```

Comment:
how about replace this description with https://daostack.io ?

Reply:
I updated the description to the same on one on the front page of the website, and added in the URL. The description field is representative of people from the outside looking in, who want to know what DAOstack is, so that they can decide if they are interested in the data.